### PR TITLE
Docs update content-2 struct

### PR DESCRIPTION
**Pull request by alvinisasi**

### CODE REVIEW

File: content/chapter-2/6-struct.md [hunk 1]
```diff
@@ -64,6 +64,7 @@ fn main() {
    println!("{}", p.get_fullname()) // "Adiatma Kamarudin"
 }
 ```
+_Block_ `impl` pada contoh di atas mendeskripsikan konteks pada `struct`. Kemudian _function_ yang direferensikan ke konteks disisipkan di dalam _block_ `impl`.
```
Comment (r17x) [inline content/chapter-2/6-struct.md:67]: block `impl` dimana ?

**Review by r17x — APPROVED** [review]

File: content/chapter-2/6-struct.md (hunk 1 above)
Comment (alvinisasi) [reply content/chapter-2/6-struct.md:67]: di bawah `struct` nya pak

Comment (r17x) [reply content/chapter-2/6-struct.md:67]: Okay mantap

**Review by adiatma — APPROVED** [review]

**Comment by r17x** [comment]
awesome @adiatma

**Pull request merged**